.infrastructure/secret.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: app-secret
+  namespace: mateapp
```

Comment:
The namespace `mateapp` does not match the namespace `todoapp` used in the Deployment and ConfigMap. Ensure that the namespace is consistent across all resources if they are intended to work together.

Reply:
done